docs/Developers/Newcomers_Guide.rst

```diff
@@ -295,6 +295,20 @@ coala project directory:
 
     $ pip3 install -r test-requirements.txt -r requirements.txt
 
+Its always best to run Pytests after running the requirement dependencies before you run coala.
```

Comment:
Change it to something like this -> `After the test dependencies have been installed run pytest to test your changes locally. `